projects/kalidea/kaligraphi/src/lib/molecules/kal-button/kal-button.component.ts

```diff
@@ -9,7 +13,46 @@ import { Component, OnInit, ViewEncapsulation, ChangeDetectionStrategy } from '@
 })
 export class KalButtonComponent implements OnInit {
 
-  constructor() { }
+  /**
+   * Is the button labelised ?
+   */
+  @Input() label: string;
+
+  /**
+   * The tab index
+   */
+  @Input() tabIndex: number;
+
+  /**
+   * Is the button disabled ?
+   */
+
+  private isDisabled = false;
+
+  @Input()
+
+  set disabled(value: boolean) {
+    this.isDisabled = coerceBooleanProperty(value);
+    this.tabIndex = this.isDisabled ? this.tabIndex : null;
```

Comment:
J'aurais réutilisé le getter :
`this.tabIndex = this.disabled ? this.tabIndex : null;`

Reply:
??